lib/ansible/modules/network/netconf/netconf_rpc.py

```diff
@@ -0,0 +1,259 @@
+
+- name: copy running to startup
+  netconf_rpc:
+    rpc: copy-config
+    content:
+      source:
+        running:
+      target:
+        startup:
+
+- name: get schema list with JSON output
+  netconf_rpc:
+    rpc: get
+    content: |
+      <filter>
+        <netconf-state xmlns="urn:ietf:params:xml:ns:yang:ietf-netconf-monitoring">
+          <schemas/>
+        </netconf-state>
+      </filter>
+    display: json
+
+- name: get schema using XML request
+  netconf_rpc:
+    rpc: "get-schema"
+    xmlns: "urn:ietf:params:xml:ns:yang:ietf-netconf-monitoring"
+    content: |
+      <identifier>ietf-netconf-monitoring</identifier>
+      <version>2010-10-04</version>
+    display: json
+"""
+
+RETURN = """
+stdout:
+  description: The raw XML string containing configuration or state data
+               received from the underlying ncclient library.
+  returned: always apart from low-level errors (such as action plugin)
+  type: string
+  sample: '...'
+stdout_lines:
+  description: The value of stdout split into a list
+  returned: always apart from low-level errors (such as action plugin)
+  type: list
+  sample: ['...', '...']
+output:
+  description: Based on the value of display option will return either the set of
+               transformed XML to JSON format from the RPC response with type dict
+               or pretty XML string response (human-readable) or response with
+               namespace removed from XML string.
+  returned: when the display format is selected as JSON it is returned as dict type, if the
+            display format is xml or pretty pretty it is retured as a string apart from low-level
+            errors (such as action plugin).
+  type: complex
+  contains:
+    formatted_output:
+      - Contains formatted response received from remote host as per the value in display format.
+"""
+try:
+    from lxml.etree import tostring
+except ImportError:
+    from xml.etree.ElementTree import tostring
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.network.netconf.netconf import dispatch
+from ansible.module_utils.network.common.netconf import remove_namespaces
+
+try:
+    import jxmlease
+    HAS_JXMLEASE = True
+except ImportError:
+    HAS_JXMLEASE = False
+
+
+def get_xml_request(module, request, xmlns, content):
+    if content is None:
+        if xmlns is None:
+            return '<%s/>' % request
+        else:
+            return '<%s xmlns="%s"/>' % (request, xmlns)
+
+    if isinstance(content, str):
+        content = content.strip()
+
+        if content.startswith('<') and content.endswith('>'):
+            # assumption content contains already XML payload
+            if xmlns is None:
+                return '<%s>%s</%s>' % (request, content, request)
+            else:
+                return '<%s xmlns="%s">%s</%s>' % (request, xmlns, content, request)
+
+        try:
+            # trying if content contains dict
+            content = eval(content)
```

Comment:
Usage of `eval` is not recommended, please use `ast.literal_eval`
Refer:
https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/network/common/utils.py#L432-L435